paasta_tools/cli/cmds/validate.py

```diff
@@ -205,16 +207,27 @@ def validate_chronos(service_path):
                 service=service, clusters=[cluster], instance_type=instance_type,
                 soa_dir=soa_dir):
             cjc = load_chronos_job_config(service, instance, cluster, False, soa_dir)
-            checks_passed, check_msgs = cjc.validate()
+            cjcs[service + '.' + cjc.get_job_name()] = cjc
```

Comment:
This looks a little more complicated than I think it needs to be. While not super efficient, can it just be:
```
for parent in get_parents()
    if parent not in get_services_for_cluster(cluster=cluster, instance_type=chronos):
        # something
```